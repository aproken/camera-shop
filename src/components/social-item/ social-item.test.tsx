import { render, RenderResult } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import SocialItem from './social-item';

test('Тестирование компонента SocialItem', () => {
  const xlinkHref = '#icon-facebook';

  const { container }: RenderResult = render(
    <MemoryRouter>
      <SocialItem xlinkHref={xlinkHref} />
    </MemoryRouter>
  );

  expect(container).toMatchSnapshot();
});
