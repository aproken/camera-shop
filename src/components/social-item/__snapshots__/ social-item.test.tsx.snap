// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Тестирование компонента SocialItem 1`] = `
<div>
  <li
    class="social__item"
  >
    <a
      aria-label="Переход на страницу "
      class="link"
      href="/#todo"
    >
      <svg
        aria-hidden="true"
        height="20"
        width="20"
      >
        <use
          xlink:href="#icon-facebook"
        />
      </svg>
    </a>
  </li>
</div>
`;
